shared/js/media/video_player.js

```diff
@@ -537,3 +577,15 @@ VideoPlayer.prototype.show = function() {
   // Call init() to show the poster
   this.controls.style.display = 'block';
 };
+
+VideoPlayer.instancesToLocalize = new WeakMap();
+
+navigator.mozL10n.ready(function() {
+  // Retrieve VideoPlayer instances by searching for container nodes.
+  for (var container of document.querySelectorAll('.video-player-container')) {
+    var instance = VideoPlayer.instancesToLocalize.get(container);
+    if (instance) {
+      instance.localize();
+    }
+  }
+});
```

Comment:
Because of the way functions work in JS, forward references are possible. Since the VideoPlayer() constructor is so long, consider putting this code up above before the constructor if you think that would make things clearer.

Also, I think that camera and gallery are the only apps to use video player, and that they both use it inside of a MediaFrame, so it occurs to me now (kind of late) that it might be cleaner to only have the ready() callback and WeakMap thing in MediaFrame, and have the media frame localize() method call the localize method of its video player.  On the other hand, that makes the component less independent.  In either case, we're phasing it out in favor of the web component that Russ is working on.